easybuild/tools/filetools.py

```diff
@@ -367,6 +367,7 @@ def search_file(paths, query, short=False, ignore_dirs=None, silent=False):
             dirnames[:] = [d for d in dirnames if not d in ignore_dirs]
 
         if hits:
+            hits.sort()
```

Comment:
it ensures that `hits` is always sorted, may be used deeper down or higher up (maybe not now, but later, who knows)

also, using `.sort()` means side-effects (`hits` is modified in-place, no explicit reassignment to `hits`), which makes it less easier to understand the code (it may be more efficient, but that'll be in the noise here, most likely)